vignettes/datafile.Rmd

```diff
@@ -15,8 +15,8 @@ knitr::opts_chunk$set(
 ```
 
 ```{r setup}
-install_packages("remotes")
-remotes::install.github("jspijker/datafile")
+install.packages("remotes", repos='https://mirror.lyrahosting.com/CRAN/')
```

Comment:
Setting the CRAN mirror is part of the configuration by the user. Some users have there own preferences for mirrors (e.g. based on their location or network, maybe even a local mirror). It is not a good idea to use a hard coded mirror.

Reply:
Looks  like more people do not configure their CRAN mirrors, oh whatever ...